### DISCUSSION TITLE
cli: provide option to discard recording entirely

### DISCUSSION THREAD

**Discussion by paololazzari** [asciinema/discussions#140] [Ideas]
Currently there are only two options available:

1. Upload recording
2. Save to /tmp

how about adding another one:

3. Discard recording

**Comment by ku1ik** [comment]
Thanks @paololazzari , I implemented your idea here: https://github.com/asciinema/asciinema/pull/576

**Comment by paololazzari** [comment]
@ku1ik Great, thank you!
